### ISSUE TITLE
fix: Creating routes/filename.dart results to Internal Server Error

### ISSUE THREAD

**Issue by cscoderr** [VeryGoodOpenSource/dart_frog#30]
**Description**
If you create hello.dart inside the routes folder and try to access localhost:8080/hello results to internal server error but when / is added to the buildHandler it works

if you create hello.dart inside routes folder this is what buildHandler looks like
```dart
Handler buildHandler() {
  const pipeline = Pipeline();
  final router = Router()
    ..all('/', routes_index.onRequest)
    ..all('hello', routes_hello.onRequest);
  return pipeline.addHandler(router);
}
```
when you try to access localhost:8080/hello an error is been throw internal server error
but if i modify it
```dart
Handler buildHandler() {
  const pipeline = Pipeline();
  final router = Router()
    ..all('/', routes_index.onRequest)
    ..all('/hello', routes_hello.onRequest);
  return pipeline.addHandler(router);
}
```

**Expected Behavior**
 / should be added new routes under buildHandler automatically
```dart
Handler buildHandler() {
  const pipeline = Pipeline();
  final router = Router()
    ..all('/', routes_index.onRequest)
    ..all('/hello', routes_hello.onRequest);
  return pipeline.addHandler(router);
}
```

**Comment by felangel** [comment]
Hi @cscoderr this should be resolved in dart_frog_cli v0.0.1-dev.4.
You should just need to upgrade via `dart pub global activate dart_frog_cli`. Let me know if that helps and sorry for the inconvenience! 👍

**Comment by cscoderr** [comment]
Hello
Thanks for reaching out
I have updated it and it helps
